server/service/integration_mdm_test.go

```diff
@@ -658,13 +666,13 @@ func checkNextPayloads(t *testing.T, mdmDevice *mdmtest.TestAppleMDMClient, forc
 	require.NoError(t, err)
 	for cmd != nil {
 		var fullCmd micromdm.CommandPayload
-		require.NoError(t, plist.Unmarshal(cmd.Raw, &fullCmd))
```

Comment:
THis is another case of the unmarshal failing when it encountered a `DeclarativeManagement` request type. Again, I moved the unmarshal to do it just when it's a known supported request type.